pokemongo_bot/cell_workers/seen_fort_worker.py

```diff
@@ -119,8 +129,7 @@ def work(self):
             elif spin_result == 2:
                 logger.log("[#] Pokestop out of range")
             elif spin_result == 3:
-                pokestop_cooldown = spin_details.get(
-                    'cooldown_complete_timestamp_ms')
+                pokestop_cooldown = userpokestop_cooldown
```

Comment:
Same thing as my other comment about ignoring pokestop cooldown the server is telling you.